### ISSUE TITLE
[ag] Proposal to supersede WCAG10

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/transitions#123]
# title, URL of the document to be superseded
[Web Content Accessibility Guidelines 1.0](https://www.w3.org/TR/WCAG10/)

# title, URL of the newer version
[Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG/)
I provide the unversioned URI for supersession, the current version is 2.1 but that will continue to evolve.

# Link to group's decision to request superseding
[RE: CFC – Marking WCAG 1.0 Superseded and related supporting documents as Retired](https://lists.w3.org/Archives/Public/w3c-wai-gl/2019JanMar/0043.html)

# Rationale
WCAG 1.0 is 20 years old and was written for the technology of the time. WCAG 2.0 was released nearly 10 years later to much fanfare with less technology-specific features and is now the version most widely referenced in policy. The release of WCAG 2.1, another 10 years on, further suggests that guidelines evolution has superseded WCAG 1.0.

# Implementation
WCAG 2.x is generally backwards compatible with WCAG 1.0. Sites that conform to WCAG 1.0 may not yet conform to WCAG 2, but are not inherently unable to do so.

**Comment by swickr** [comment]
Approved.  Michael, please include me in the preparation of the draft AC CfR.

**Comment by plehegar** [comment]
@@TODO: check if WCAG 1.0 is currently used in a country...

**Comment by michael-n-cooper** [comment]
AC survey created https://www.w3.org/2002/09/wbs/33280/2019-04_WCAG10_supersede/

**Comment by plehegar** [comment]
awaiting on comm draft.

**Comment by michael-n-cooper** [comment]
I didn't do the comm draft because Judy told me to hold the presses, something was terribly wrong with the template I was asked to follow, and not to do anything until it was sorted out, which wouldn't be this week. According to her, @plehegar knows about this and what's wrong with the template. I don't. So I'm awaiting you guys before I do the comm draft.

**Comment by michael-n-cooper** [comment]
Looks like I dropped the ball after the "hold the presses". PLH created a second AC survey: https://www.w3.org/2002/09/wbs/33280/wcag1-sup/ which has 1 response so we'll use that one. PLH to send draft comm announcement to me and Judy for review.

**Comment by plehegar** [comment]
draft announcement to start AC review sent to Michael and Judy.

**Comment by michael-n-cooper** [comment]
Removing from my plate, afaik the success or failure of this task is on Judy's plate now.

**Comment by plehegar** [comment]
https://www.w3.org/TR/1999/WAI-WEBCONTENT-19990505/
